facia-tool/public/js/modules/content-api.js

```diff
@@ -186,13 +195,18 @@ function decorateBatch (articles) {
         }
 
         results.forEach(function(result) {
-            var icc = internalContentCode(result);
+            var pageCode = internalPageCode(result),
+                icc = internalContentCode(result);
 
-            if(icc) {
-                cache.put('contentApi', icc, result);
+            if (pageCode) {
+                cache.put('contentApi', pageCode, result);
+                if (icc) {
```

Comment:
so actually it will use the new code in preference but the old code if it's not present right? is that based on whether the content was created in R2 or composer, or is it going to change over completely at some point?